api/stats/index.js

```diff
@@ -3,14 +3,79 @@
 let request = require("leo-auth");
 let stats = require("../../lib/stats.js");
 require("moment-round");
+var zlib = require("zlib");
+
+let compressionThreshold = 100000; // 100k
 
 exports.handler = require("leo-sdk/wrappers/resource")(async (event, context, callback) => {
     await request.authorize(event, {
         lrn: 'lrn:leo:botmon:::',
         action: "stats",
         botmon: {}
     });
+
+    function findBigStrings(obj, prefix = "") {
```

Comment:
@jgrantr Do you know why this was added?

Reply:
@czirker I think this was to track do-nothings that were too big.

Reply:
We will leave it in for now